src/js/components/inspectors/Property.jsx

```diff
@@ -60,6 +73,9 @@ var Property = React.createClass({
         field = props.field,
         value = state.value,
         disabled = props.disabled || props.group,
+        onChange = props.onChange || this.handleChange,
+        onBlur = props.onBlur,
+        docId = props.id,
```

Comment:
@YiyangXu and I added this to avoid future merge conflicts, I believe somewhere in her ScaleInspector component she was using an id for dom manipulation; will speak to her again about the need for docId.